kubehub/views/read_deploy_log_file.py

```diff
@@ -0,0 +1,26 @@
+from json import loads
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+
+from kubehub_backend.settings import K8S_DEPLOY_LOG_DIR
+from ..models.kubespray_deploy import KubesprayDeploy
+
+
+@csrf_exempt
+def read_deploy_log_file(request):
+    if request.method == 'POST':
+        try:
+            data = loads(request.body)
+            kubespray_deploy_id = KubesprayDeploy.objects.get(pk=data["kubespray_deploy_id"]).id
+            filename = f"kubespray_deploy_{kubespray_deploy_id}.log"
```

Comment:
expose to separate function, measure perfomance

Reply:
fixed in b8f5308 used function with higher performance

https://docs.google.com/document/d/1RKLQ4_aaDyKWg83hcn0oE5-qUuv11aLPk_pacI-_3W0/edit?usp=sharing